prin

```diff
@@ -142,15 +144,15 @@ if [[ ${POSITIONAL_ARGV[0]} =~ [0-9]{5,} ]]; then
   # non-cherrypick commit searches on branches
   INITIAL_COMMIT=$(git log --grep "Merge pull request #$PR" \
                            --all --pretty=format:"%h")
-  CP_COMMITS=($(git log --grep "cherry-pick-of-.*#$PR-" \
-                        --all --pretty=format:"%h"))
+  mapfile -i CP_COMMITS < <(git log --grep "cherry-pick-of-.*#$PR-" \
+                        --all --pretty=format:"%h")
```

Comment:
@pswica, here I met this error when used `mapfile  -i` :
`/Users/llhu/Desktop/test/release/release/prin: line 154: mapfile: -i: invalid option
mapfile: usage: mapfile [-d delim] [-n count] [-O origin] [-s count] [-t] [-u fd] [-C callback] [-c quantum] [array]`
Is there some errors in my local side. Thanks

Reply:
Thanks os much for catching. No this was my bad. I messed up by putting `-i` instead of `-t`

Reply:
PR here